test/e2e/antctl_test.go

```diff
@@ -122,7 +122,7 @@ func testAntctlAgentLocalAccess(t *testing.T, data *TestData) {
 		cmd := strings.Join(args, " ")
 		t.Run(cmd, func(t *testing.T) {
 			stdout, stderr, err := runAntctl(podName, args, data)
-			if err != nil && !strings.HasSuffix(stderr, "not enabled\n") {
+			if err != nil && !(strings.HasSuffix(stderr, "not enabled\n") || strings.HasSuffix(stderr, "there is no effective bgp policy applied to the Node\n")) {
```

Comment:
I understand the change, but we need to find a better way to test this
The test is quite shallow anyway, as it doesn't really validate much, but anyway that is out-of-scope of the PR